publishable/database/migrations/2016_01_01_000000_add_user_avatar_and_role_id.php

```diff
@@ -6,25 +6,23 @@ class AddUserAvatar extends Migration
 {
     /**
```

Comment:
well, uninstalling packages should not touch the db in anyway, because what happens if the user has added new data ?
also if the user is going to uninstall it, it will mostly be in testing phase which 99% wont care what happens with the db tables.